vsts/pipelines/templates/_cleanImageCacheTemplate.yml

```diff
@@ -4,3 +4,8 @@ steps:
     docker system prune -af
     docker ps -a
   displayName: 'Remove all existing docker images from machine'
+
+- script: |
```

Comment:
its good to reboot  our build machines  at least once a week .. currently its every saturday at 2:30 am